l10n/en/products/vpn/shared.ftl

```diff
@@ -0,0 +1,77 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/
+
+vpn-shared-product-name = { -brand-name-mozilla-vpn }
+vpn-shared-subscribe-link = Get { -brand-name-mozilla-vpn }
+vpn-shared-waitlist-link = Join the Waitlist
+vpn-shared-sign-in-link = Already a subscriber?
+vpn-shared-available-countries = We currently offer { -brand-name-mozilla-vpn } in the US, the UK, Canada, New Zealand, Singapore and Malaysia.
+vpn-shared-available-countries-v2 = We currently offer { -brand-name-mozilla-vpn } in the US, the UK, Germany, France, Canada, Malaysia, New Zealand, and Singapore.
+vpn-shared-availablity-long = The VPN is currently available for { -brand-name-windows } 10, { -brand-name-android } and { -brand-name-ios } devices in the US, the UK, Canada, New Zealand, Singapore and Malaysia.
+vpn-shared-money-back-guarantee = 30-day money-back guarantee
+
+# This string will be followed by a lockup of press logos for publications that have featured Mozilla VPN.
+vpn-shared-featured-in = Featured in
+
+vpn-shared-features-encrypt = Device-level encryption
+
+# Variables:
+#   $servers (number) - number of available servers
+#   $countries (number) - number of available countries
+vpn-shared-features-servers = { $servers }+ servers in { $countries }+ countries
+
+vpn-shared-features-bandwidth = No bandwidth restrictions
+vpn-shared-features-activity = No logging of your network activity
+
+# Variables:
+#   $devices (number) - maximum number of connected devices
+vpn-shared-features-devices = Option to connect up to { $devices } devices
+
+# Variables:
+#   $servers (number) - number of available servers
+#   $countries (number) - number of available countries
+vpn-shared-features-server-countries = Connect to more than { $servers } servers in over { $countries } countries
```

Comment:
nit/copy: Should there be a `+` after each variable?

Reply:
The copy says "more than..." and "in over..." so omiting the `+` is OK here I think.